SConstruct

```diff
@@ -623,15 +642,15 @@ if "help" in COMMAND_LINE_TARGETS:
 
         if list:
             # show formatted list of options
-            logger.info("\nConfiguration options for building Cantera:", print_level=False)
-            logger.info(config.list_options(), print_level=False)
+            logger.status("\nConfiguration options for building Cantera:\n", print_level=False)
+            logger.status(config.list_options() + "\n", print_level=False)
```

Comment:
I think that most `INFO` level messages make most sense for `scons build`, but clutter output otherwise. I created status so we can trigger output at the `WARNING` level with a more appropriate nomenclature.